tasks/2_convert-to-f/convert-to-f.js

```diff
@@ -0,0 +1,6 @@
+export function convertToF(celsius) {
+  var fahrenheit;
+  fahrenheit = celsius * 9 / 5;
+  fahrenheit +=32;
+  return fahrenheit;
```

Comment:
Or just return expression, it's up to you

Reply:
i chose const

Reply:
@miffn3 why do you need to introduce const here? :)

Reply:
> @miffn3 why do you need to introduce const here? :)

because this value doesn't change)) My bad)